buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -306,9 +321,33 @@ public void plugin(String pluginProjectPath) {
     @Override
     public void module(Provider<RegularFile> module) {
         checkFrozen();
+        registerExtractedConfig(module);
         this.modules.add(module.map(RegularFile::getAsFile));
     }
 
+    private void registerExtractedConfig(Provider<RegularFile> pluginProvider) {
+        if (bundleAttribute == null) {
+            bundleAttribute = Attribute.of("bundle", Boolean.class);
```

Comment:
Should we just initialize this in the constructor then?

Reply:
changed